tests/.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  env: {
+    embertest: true
+  },
+  globals: {
+    capture: true,
+    chai: true,
```

Comment:
These entries were in the `.eslintrc` file so I moved them to the `.eslintrc.js` file as part of the upgrade.

Whether we want them to remain or not is technically not really part of this PR and should be done as a separate effort.

That being said, it's easy enough to not add them right now, so we just need to come to a consensus.  I did not look at what other addons were doing because I assumed, perhaps erroneously, that things were already configured how we wanted them, because again, I was not here to make changes to configuration, only upgrade `ember-cli`